tests/acceptance/test_update_control.py

```diff
@@ -492,7 +492,7 @@ def test_update_control_maps(
                 # boot expiration mechanism.
                 connection.run("systemctl restart mender-client")
 
-            now = time.time()
+            now = qemu_system_time(connection)
```

Comment:
Once we are here, we could rename `now` to something more meaningful like `test_start_time` or `update_start_time`...

Reply:
Yes - agreed! Let's. I'll add a separate commit for this.